docs/tutorials/gcp_gpu.md

```diff
@@ -176,8 +181,8 @@ to set up NVIDIA Container Toolkit.
 
 ## 9. Install NVIDIA cuQuantum Appliance (Option 3 only)
 Follow the instructions [here](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cuquantum-appliance)
-to set up cuQuantum Appliance. You may need to use `sudo` for the Docker commands.
-
+to set up cuQuantum Appliance. If this link is broken, you can try the [cached page](https://webcache.googleusercontent.com/search?q=cache%3Ahttps%3A%2F%2Fcatalog.ngc.nvidia.com%2Forgs%2Fnvidia%2Fcontainers%2Fcuquantum-appliance&rlz=1CAFOSO_enUS1067&oq=cache%3Ahttps%3A%2F%2Fcatalog.ngc.nvidia.com%2Forgs%2Fnvidia%2Fcontainers%2Fcuquantum-appliance&gs_lcrp=EgZjaHJvbWUqBggAEEUYOzIGCAAQRRg7MgYIARBFGDoyBggCEEUYPDIGCAMQRRg80gEIMzMyMGowajSoAgCwAgA).
+You may need to use `sudo` for the Docker commands.
```

Comment:
This is not appropriate, I'd say. If a link from NVIDIA is broken, what's expected is to report to NVIDIA (as you've kindly done - https://github.com/NVIDIA/cuQuantum/issues/118 - thank you!) and wait for resolution. Most likely it's transient and will be fixed shortly, unless it's because NVIDIA decides to discontinue the cuQuantum Appliance support, in which case we should nuke this section and look for alternatives. If we were to add a cache link to every single external page, it is unmaintainable. 

cc: @tlubowe @mtjrider for vis

Reply:
I removed that sentence, thanks.

Reply:
Thank you!